### PR TITLE
Wip theming

### PR DESCRIPTION
**Pull request by David-Corbett**
some more incremental changes - pay attention to the preprocess_page hook

### CODE REVIEW

File: src/js/toggle-btn-script.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  // eslint-disable-next-line
+  var $relatedArticleList = $(".toggle-list ul"),
+    maxRelated =
+      parseInt($(".toggle-list").attr("data-toggle-length"), 10) || 7,
+    relatedTotal = $relatedArticleList.children("li").length,
+    // eslint-disable-next-line
+    relatedToggleText = "Show " + (relatedTotal - maxRelated) + " more";
+
+  if (relatedTotal > maxRelated) {
+    // $relatedArticleList.setAttribute('id', 'toggle-menu');
```
Comment (omahm) [inline src/js/toggle-btn-script.js]: Any commented out code should be removed.

**Review by omahm — COMMENTED** [review]
Looks good. Only real issue is with the comments and to pass code standards they should start with a capital and end with a full stop.

File: src/js/toggle-btn-script.js (hunk 1 above)
Comment (David-Corbett) [reply src/js/toggle-btn-script.js]: that line should have actually have been removed, so have done this as part of the PR

**Review by omahm — APPROVED** [review]
LG

**Pull request merged**